packages/zk-certificates/tasks/reissueZkCertificate.ts

```diff
@@ -156,15 +177,22 @@ task(
     false,
   )
   .addParam(
-    'kycDataFile',
+    'zkCertificateDataFile',
```

Comment:
Can you please also shorten the parameter name here to be consistent with the createZkCertificate task?